src/services/projects.js

```diff
@@ -92,6 +92,18 @@ module.exports = class ProjectsHelper {
 				updated_by: loggedInUserId,
 			}
 
+			if (
+				orgConfig &&
+				orgConfig?.result?.config &&
+				orgConfig?.result?.config?.external_resource_visibility_policy
+			) {
+				//get visiblity and related_org details
+				const result = await this.populateVisibilityAndRelatedOrgs(projectData, orgConfig, orgCode, tenantCode)
```

Comment:
@praveenKDass write only one function in utils to return the policy keys, no need to write the same function in program, project